docs/key_features/async_activate_offload.md

```
@@ -0,0 +1,252 @@
+
+Offload targets are automatically derived from `model._no_split_modules` — the same
+class names used for FSDP sharding boundaries. This mirrors the FSDP enablement pattern
+where `_no_split_modules` determines sharding units automatically.
+For example, with Qwen3-MoE where `_no_split_modules = ["Qwen3MoeDecoderLayer"]`,
+all `Qwen3MoeDecoderLayer` instances are automatically identified as offload targets.
+
+## Configuration Reference
+
+| Parameter | Type | Default | Description |
+| --- | --- | --- | --- |
+| `train.accelerator.offload_config.enable_async_activation_offload` | bool |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)